.github/workflows/test.yaml

```
@@ -45,6 +45,11 @@ jobs:
       - name: Lint
         run: yarn lint
 
+      - name: Unit test stylelint-plugin
+        run: |
+          yarn --cwd stylelint-plugin install --frozen-lockfile
+          yarn --cwd stylelint-plugin test:unit
```

Dependencies are resolved from the root
```suggestion
          yarn test:unit
```